installing-from-source.rst

```diff
@@ -53,8 +53,11 @@ Now activate the virtualenv in your current shell session:
 
     # For bash/zsh users:
     source ./venv/bin/activate
+    # For git bash users:
```

Comment:
Please specify "windows git bash" to be more clear so someone doesn't get confused.
If I was a exclusive mac/linux user I wouldn't know what "git bash" was and probably be very confused.